pkg/validation/commands.go

```diff
@@ -82,7 +83,13 @@ func validateGroup(commands []v1alpha2.Command) error {
 	if defaultCommandCount == 0 {
 		return fmt.Errorf("there should be exactly one default command, currently there is no default command")
 	} else if defaultCommandCount > 1 {
-		return fmt.Errorf("there should be exactly one default command, currently there is more than one default command")
+		var commandsReference string
+		for _, command := range defaultCommands {
+			commandsReference += resolveErrorMessageWithImportAttributes(fmt.Errorf("; command: %s", command.Id), command.Attributes).Error()
+		}
```

Comment:
nice, we can also do the same logic for the above if block `if defaultCommandCount == 0 {`

Reply:
IMO, we don't need to do for `defaultCommandCount == 0`, as nothing will be needed to print out.